akka-actor/src/main/scala/akka/event/DeadLetterListener.scala

```diff
@@ -26,21 +32,72 @@ class DeadLetterListener extends Actor {
   override def postStop(): Unit =
     eventStream.unsubscribe(self)
 
-  def receive = {
-    case DeadLetter(message, snd, rcp) =>
+  private def incrementCount(): Unit = {
+    // `count` is public API (for unknown reason) so for backwards compatibility reasons we
```

Comment:
So, we expect that people extend and adapt this class?

Reply:
it shouldn't have been public in the first place, but now it is